recipes/hyper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hyper" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "069514f54231fb7b5df2fb910a114663a83306d5296f588fffcb0a9be19407fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
This one needs <code>enum34>=1.0.4,<2 &nbsp;# [py2k]</code> too.

You also might want to consider packaging the optional dependency [pycohttpparser](https://pypi.python.org/pypi/pycohttpparser/1.1.0), but can do that later if you want.

Reply:
Note: `enum34` done; pycohttpparser can be done later.